### ISSUE TITLE
Accessibility labels on Search screen 

### ISSUE THREAD

**Issue by weglmar** [StatCan/StatcanPublicationUAT#514]
#311 

On the Search screen: 
The following button labels are still read in English, after changing the language in French:

Search – “Clear text” button should be labelled as “Effacez le texte” bouton
Search – Sort by options should be labelled as “trier par”

**Comment by weglmar** [comment]
@ForumG-SC Hi Forum, when I try to select the Clear text button, it's only selecting Annuler and reading that out.

**Comment by weglmar** [comment]
Seems to be resolved, but it is reading the French buttons as English (ex. Effacez le texte button, not Effacez le texte bouton). I'm unsure if this can be read in French? I have noticed this with other buttons too. 
@Rajat-Sen, could you please confirm?

**Comment by Rajat-Sen** [comment]
Since your device language is set to English and you change the language to French in the app, that's why there are accent issues. Otherwise, it should be fine.

The original issues have been fixed with correct labels. New tickets will need to be created for the screen-flow.  We can close this one for now.

**Comment by weglmar** [comment]
Closing as the issue in this ticket has been resolved.
